prime-router/src/main/kotlin/azure/BlobAccess.kt

```diff
@@ -55,6 +58,31 @@ class BlobAccess() : Logging {
         }
     }
 
+    /**
+     * Data structure for configuring reusable blob store container access.
+     */
+    data class BlobContainerMetadata(
```

Comment:
This was formerly the class used to uniquely identify container clients for reusability, but the existing properties were exactly what we needed for the purpose of defining a blob store to access, so the class is now exposed to other methods in RS that may use BlobAccess. The build functions serve to provide a connection string resolver in cases where it needs to be resolved from environment variable.